app/assets/javascripts/app/components/button/button.js

```diff
@@ -27,7 +27,10 @@
   function councilRaisedButton() {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        disabled: '=',
+        click: '&'
+      },
```

Comment:
I'm not really doing anything with them, just passing to the html, I think it keeps it simple